### PR TITLE
Bugfix/earn page blinking

### PR DESCRIPTION
**Pull request by phucvarmeta**

### CODE REVIEW

File: packages/web/src/components/common/my-position-card-list/MyPositionCardList.tsx [hunk 1]
```diff
@@ -38,8 +38,8 @@ const MyPositionCardList: React.FC<MyPositionCardListProps> = ({
   themeKey,
   showLoadMore,
 }) => (
-  <CardListWrapper loading={isLoading}>
-    <GridWrapper ref={divRef} onScroll={onScroll} loading={isLoading}>
+  <CardListWrapper $loading={isLoading}>
```
Comment (jinoosss) [inline packages/web/src/components/common/my-position-card-list/MyPositionCardList.tsx:41]: Why did you change it to `$loading`?
Is it to express a property of the styled component?

File: packages/web/src/components/earn/incentivized-pool-card/IncentivizedPoolCard.tsx [hunk 2]
```diff
@@ -18,16 +17,17 @@ export interface IncentivizedPoolCardProps {
   pool: PoolCardInfo;
   routeItem: (id: string) => void;
   themeKey: "dark" | "light";
+  isStakedPool: (poolPath: string | null) => boolean
```
Comment (jinoosss) [inline packages/web/src/components/earn/incentivized-pool-card/IncentivizedPoolCard.tsx:20]: It's confusing that `isStakedPool` is a function. 
It's certainly not misnamed, but we already use boolean types for `isLoading`, `isPoolFetched`, etc. 
How about changing it to `checkStakedPool`?

File: packages/web/src/constants/common.constant.ts [hunk 3]
```diff
@@ -1 +1,5 @@
 export const DEFAULT_NETWORK_ID = "dev.gnoswap";
+
+export const PATH = ["/earn"];
+export const PATH_10SECOND = ["/earn/pool/[pool-path]/remove", "/tokens/[token-path]"];
```
Comment (jinoosss) [inline packages/web/src/constants/common.constant.ts:4]: Can you let me know the meaning of SECOND?

File: packages/web/src/containers/earn-my-position-container/EarnMyPositionContainer.tsx [hunk 4]
```diff
@@ -43,7 +43,7 @@ const EarnMyPositionContainer: React.FC<
 
     const { connected, connectAdenaClient, isSwitchNetwork, switchNetwork, account } = useWallet();
     const { updateTokenPrices } = useTokenData();
-    const { updatePositions, isFetchedPools, loading } = usePoolData();
+    const { updatePositions, isFetchedPools, loading: loadingPool } = usePoolData();
```
Comment (jinoosss) [inline packages/web/src/containers/earn-my-position-container/EarnMyPositionContainer.tsx:46]: What about `isLoading` or `isLoadingPool` instead of `loadingPool`?

File: packages/web/src/containers/incentivized-pool-card-list-container/IncentivizedPoolCardListContainer.tsx [hunk 5]
```diff
@@ -35,12 +35,11 @@ const IncentivizedPoolCardListContainer: React.FC = () => {
   const [page, setPage] = useState(1);
   const router = useRouter();
   const [mobile, setMobile] = useState(false);
-  const { incentivizedPools, isFetchedPools, updatePools, loading } = usePoolData();
+  const { incentivizedPools, isFetchedPools, updatePools, loading: loadingPool } = usePoolData();
```
Comment (jinoosss) [inline packages/web/src/containers/incentivized-pool-card-list-container/IncentivizedPoolCardListContainer.tsx:38]: What about `isLoading` or `isLoadingPool` instead of `loadingPool`?

File: packages/web/src/hooks/common/use-loading.tsx [hunk 6]
```diff
@@ -1,41 +1,29 @@
...
 import { useEffect, useState } from "react";
 
-interface Props {
-  isLoading: boolean;
-  isFetching: boolean;
-  isBack: boolean;
-  status: boolean;
-  connected: boolean;
+interface UseLoadingProps {
+  shouldTrigger: boolean;
```
Comment (jinoosss) [inline packages/web/src/hooks/common/use-loading.tsx:4]: `shouldTrigger` feels like the name of a function.
Is there a better name ?

**Review by jinoosss — COMMENTED** [review]
Great work. 
I left a few comments.

File: packages/web/src/constants/common.constant.ts (hunk 3 above)
Comment (phucvarmeta) [reply packages/web/src/constants/common.constant.ts:4]: i inherited the previous version of code base, it means that on the pages that represent by those paths the function have 10s time out for refetch interval

Comment (phucvarmeta) [reply packages/web/src/constants/common.constant.ts:4]: it mean the `useGetPositionsByAddress` on those paths will continuously refetch after each 10 second

Comment (jinoosss) [reply packages/web/src/constants/common.constant.ts:4]: By time, you meant data refetching cycles.
I'd appreciate it if you could do that in the comments as well.

File: packages/web/src/containers/earn-my-position-container/EarnMyPositionContainer.tsx (hunk 4 above)
Comment (phucvarmeta) [reply packages/web/src/containers/earn-my-position-container/EarnMyPositionContainer.tsx:46]: i will fix this

File: packages/web/src/containers/incentivized-pool-card-list-container/IncentivizedPoolCardListContainer.tsx (hunk 5 above)
Comment (phucvarmeta) [reply packages/web/src/containers/incentivized-pool-card-list-container/IncentivizedPoolCardListContainer.tsx:38]: i will fix this

File: packages/web/src/hooks/common/use-loading.tsx (hunk 6 above)
Comment (phucvarmeta) [reply packages/web/src/hooks/common/use-loading.tsx:4]: i will fix this

File: packages/web/src/components/common/my-position-card-list/MyPositionCardList.tsx (hunk 1 above)
Comment (phucvarmeta) [reply packages/web/src/components/common/my-position-card-list/MyPositionCardList.tsx:41]: this is a fix for style component warning, it's a transition props
please check for more detail: https://styled-components.com/docs/faqs#why-am-i-getting-html-attribute-warnings

File: packages/web/src/hooks/common/use-loading.tsx (hunk 6 above)
Comment (phucvarmeta) [reply packages/web/src/hooks/common/use-loading.tsx:4]: @jinoosss i do some research and i think that this name is not bad at all, can i use it

Comment (phucvarmeta) [reply packages/web/src/hooks/common/use-loading.tsx:4]: I think i will change it to `shouldTriggerLoading` instead, is it good ?

File: packages/web/src/components/earn/incentivized-pool-card/IncentivizedPoolCard.tsx (hunk 2 above)
Comment (phucvarmeta) [reply packages/web/src/components/earn/incentivized-pool-card/IncentivizedPoolCard.tsx:20]: i will fix this

File: packages/web/src/constants/common.constant.ts (hunk 3 above)
Comment (gotecq-phuctran) [reply packages/web/src/constants/common.constant.ts:4]: Yes, It a specific time for data refetching cycles, I will notice about the confusing code in the future.

File: packages/web/src/hooks/common/use-loading.tsx (hunk 6 above)
Comment (jinoosss) [reply packages/web/src/hooks/common/use-loading.tsx:4]: In my opinion, `trigger` is sufficient. 
Because `trigger` is a boolean type.
If `trigger` is true, we think the `trigger` has been fired.

In that case, please change the function of `triggerLoading` to `startLoading`.

**Review by jinoosss — DISMISSED** [review]
Nicely done.

**Review by jinoosss — APPROVED** [review]
